src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -0,0 +1,158 @@
+        return temp
+    })
+    const intervalRef = useRef(null)
+
+    const [sorting, setSorting] = React.useState<SortingState>([])
+
+    const region = useMemo(() => {
+        return is_eu
+            ? 'https://deriv-static-pricingfeed.firebaseio.com/eu.json'
+            : 'https://deriv-static-pricingfeed.firebaseio.com/row.json'
+    }, [is_eu])
+
+    const getData = useCallback(async () => {
+        const rawResponse = await fetch(region)
+        const response = await rawResponse.json()
+        setRawMarketsData(response)
+    }, [region])
+
+    useEffect(() => {
+        if (!initial_loaded) {
+            getData()
+            setInitialLoaded(true)
+        }
+        intervalRef.current = setInterval(getData, 10000)
+        return () => {
+            clearInterval(intervalRef.current)
+        }
+    }, [getData, initial_loaded])
+
+    const updateData = useCallback(() => {
+        if (rawMarketsData) {
+            const stocks = rawMarketsData.market['stocks']
+            const indices = rawMarketsData.market['indices']
+            const stocks_indices = Object.assign(stocks, indices)
+            const res = { ...rawMarketsData.market, indices: { ...stocks_indices } }
+
+            Object.keys(res).map((item) => {
+                if (item == selected_market) {
+                    const selected_market_data = res[item]
+                    const result = Object.values(selected_market_data)
+                    setMarketsData(result)
+                }
+            })
+        }
+    }, [rawMarketsData, selected_market])
+
+    useEffect(() => {
+        updateData()
+    }, [updateData])
+
+    const columns = useLiveColumns()
+
+    const table = useReactTable({
+        data: markets_data,
+        columns,
+        state: {
+            sorting,
+        },
+        getCoreRowModel: getCoreRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        onSortingChange: setSorting,
+    })
+
+    const rows = table.getRowModel().rows.slice(0, TABLE_VISIBLE_ROWS)
+
+    return (
+        <>
+            <Flex.Box justify="center" mt="16x">
+                <table>
+                    <thead>
+                        {table.getHeaderGroups().map((headerGroup) => (
+                            <Flex.Box key={headerGroup.id} justify="center">
+                                <tr className={table_row_header}>
+                                    {headerGroup.headers.map((header) => (
+                                        <th key={header.id}>
+                                            {header.isPlaceholder
+                                                ? null
+                                                : flexRender(
+                                                      header.column.columnDef.header,
+                                                      header.getContext(),
+                                                  )}
```

Comment:
https://react.dev/learn/conditional-rendering#recap
```suggestion
                                            {!header.isPlaceholder && flexRender(
                                                      header.column.columnDef.header,
                                                      header.getContext(),
                                                  )}
```